boards/qemu/riscv32/board.yml

```diff
@@ -5,3 +5,4 @@ board:
     - name: qemu_virt_riscv32
       variants:
         - name: smp
+        - name: opensbi
```

Comment:
You should add the instructions you provided here: https://github.com/zephyrproject-rtos/zephyr/issues/68133#issuecomment-2304523249 to this board's documentation